packages/@aws-cdk/aws-codepipeline-actions/test/ecs/ecs-deploy-action.test.ts

```diff
@@ -197,6 +199,99 @@ describe('ecs deploy action', () => {
 
     });
   });
+  describe('ECS deploy Action service reference cross account/region', () => {
```

Comment:
Can we actually move this to a separate file, and make this an integ test? If we change its extension to `*.lit.ts`, we'll be able to reference it from the ReadMe of this module.

We have examples of this in this module already, for example [this](https://github.com/aws/aws-cdk/blob/ef8ab7245630f54b5a633925e27b34c31b712abc/packages/%40aws-cdk/aws-codepipeline-actions/test/integ.lambda-deployed-through-codepipeline.lit.ts#L1-L1).

We also have examples of integ tests with multiple Stacks, like [here](https://github.com/aws/aws-cdk/blob/ef8ab7245630f54b5a633925e27b34c31b712abc/packages/%40aws-cdk/aws-apigateway/test/integ.restapi.multistack.ts#L1).

Now, I assume the integ test for this will not deploy, because our integ tests only run in one account. That's fine - you can generate the expected JSON file for these running the command `yarn integ --dryrun your-integ-file.lit.js`. Let's add a comment in the code of the integ test that says it doesn't deploy, and why.

Reply:
Should we do 2 files, since they are really 2 different examples?

One for `Deploying ECS Deploy Action of existing service across accounts and regions` and one for `Deploying ECS Deploy Action of new service across accounts and regions`.

Reply:
Yep! Sorry, I missed that there are 2 separate examples.

Reply:
Changed to using the lit files and updated the ReadMe.